docs/manual/article-management/simple-tokens.de.md

```diff
@@ -0,0 +1,90 @@
+Simple Tokens sind Platzhalter, die ebenso wie Insert-Tags, die Inhalte ausgeben können. Simple Tokens beschränken sich jedich nur auf die Inhalte des jeweiligen Datenbank-Feldes. Der Unterschied ist jedoch, dass Simple Tokens nicht nur Inhalte des jeweiligen Datenbank-Feldes ausgegeben werden können, sondern auch Fallabfragen und reguläre Ausdrücke überprüft und ausgegeben werden können. Dank der Symfony Expression Language können die Einsatzzwecke noch erweitert werden.
```

Comment:
Das Hauptproblem mit diesem Artikel ist für mich, dass Simple Tokens ein generelles Konzept sind. Sie werden hier allerdings so beschrieben, als würden sie sich direkt auf irgendwelche Datenbank-Felder beziehen. Das ist aber keineswegs der Fall. Simple Tokens sollten m.M.n. als generelles Kapitel erklärt werden. Welche Tokens dabei zur Verfügung stehen, entscheidet der/die Entwickler/in. Ob es Tokens sind, die Datenbankfelder repräsentieren oder Formulardaten oder was auch immer, ist völlig egal für die Erklärung des generellen Konzepts :-)

Reply:
Dieser Einleitungstext wurde mittlerweile schon von jemanden bearbeitet.